examples/plot_derivative.py

```diff
@@ -0,0 +1,79 @@
+"""
+Derivatives
+===========
+This example demonstrates how to use a set of derivative operators, namely
+:py:class:`pylops_mpi.basicoperators.MPIFirstDerivative` and
+:py:class:`pylops_mpi.basicoperators.MPISecondDerivative`.
+
+The derivative operators are really useful when dealing with
+inverted models.
+
+"""
+from matplotlib import pyplot as plt
+import numpy as np
+from mpi4py import MPI
+
+import pylops_mpi
+
+plt.close("all")
+np.random.seed(42)
+
+rank = MPI.COMM_WORLD.Get_rank()
+size = MPI.COMM_WORLD.Get_size()
+
+###############################################################################
+# Let’s start by applying the first derivative using a :py:class:`pylops_mpi.DistributedArray`
+# in the first direction i.e. along ``axis=0`` using the
```

Comment:
`in the first direction i.e. along ``axis=0``` -> `along the first direction (i.,e. along ``axis=0``)`

Reply:
Done